compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangParserListener.java

```diff
@@ -2065,6 +2065,260 @@ public void exitStringTemplateLiteral(BallerinaParser.StringTemplateLiteralConte
         this.pkgBuilder.createStringTemplateLiteral(getCurrentPos(ctx), getWS(ctx), stringFragments, endingText);
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void enterTableQueryExpression(BallerinaParser.TableQueryExpressionContext ctx) {
+        if (ctx.exception != null) {
+            return;
```

Comment:
Return is not required here. And what is the purpose of overriding this method here

Reply:
Removed the method.